dashed/assets/javascripts/dashboard.js

```diff
@@ -113,6 +113,16 @@ var Dashboard = function (dashboardData) {
           };
         }
       }).data('gridster');
+
```

Comment:
it's much much more performant to do this with CSS and not JS, I had been using JS because click events are harder to do with CSS.